creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -3,7 +3,15 @@ const axios = require('axios')
 const { logger } = require('../logging')
 
 // TODO: config var
-const PEER_HEALTH_CHECK_REQUEST_TIMEOUT = 2000
+// Can adjust as necessary
+const PEER_HEALTH_CHECK_REQUEST_TIMEOUT = 2000 // ms
+const MINIMUM_STORAGE_PATH_SIZE = 100000000000 // bytes; 100GB
```

Comment:
we'll unfortunately definnitely need to make most of these envvars since they will be very different from local to staging to prod, eg local and staging will never have 100GB free storage. also won't have 5000 30day rolling sync. will need to consider the edge case where the node only has 3-5 days of data that it returns for the 30day, ie it was just restarted